### PR TITLE
Homework2: Victoria Kandrashova

### PR DESCRIPTION
**Pull request by Vict8ria**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle)
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks
- [x] my PR made from the fork feature branch to maxim homework# branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore

### CODE REVIEW

File: src/homework2/task3.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"""
+    Task 3. Вводится строка.
+    Требуется удалить из нее повторяющиеся символыи все пробелы.
+"""
+
+
+str_ = "abc cde def"
+unique_str = ""
+
+for substring in str_:
```
Comment (belovmd) [inline src/homework2/task3.py]: symbol would be better name

File: src/homework2/task7.py [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    [4, 3, 2, 5] would return [4, 3, 2, 6]
+"""
+
+
+def up_array(arr):
+    if not len(arr):
+        return None
+
+    for i in arr:
+        if i < 0 or i > 9:
```
Comment (belovmd) [inline src/homework2/task7.py:172]: 0 < i < 9

**Review by belovmd — CHANGES_REQUESTED** [review]
good job
please address these minor issues

File: src/homework2/task7.py (hunk 2 above)
Comment (Vict8ria) [reply src/homework2/task7.py:172]: I've tested it in codewars and in this case operation 0 < i < 9 doesn't work

this example can work:
if 0 <= i <= 9:
    continue
else:
    return None

but it looks better in this case when "if" operation writes without else, that's why I used construction "i < 0 or i > 9"

Comment (belovmd) [reply src/homework2/task7.py:172]: oh sorry I've missed it's (-inf; 0) + (9, inf), not (0, 9)

**Review by belovmd — COMMENTED** [review]
9

**Pull request closed without merging**